source/retryable-writes/retryable-writes.rst

```diff
@@ -274,8 +275,8 @@ This includes an `update`_ command where any statement in the updates sequence
 specifies a ``multi`` option of ``true`` or a `delete`_ command where any
 statement in the ``deletes`` sequence specifies a ``limit`` option of ``0``. In
 the context of the `CRUD`_ specification, this includes the ``updateMany()`` and
-``deleteMany()`` methods. Drivers MUST rely on the server to raise an error if
-multi-document write operations are not supported. 
+``deleteMany()`` methods. Drivers MUST rely on the server to raise an error if a
+multi-document write operations would be retried and is not supported.
```

Comment:
I have a TODO item below in the spec where I'd like to discuss exactly how we handle these unsupported write operations. Please use that comment thread for discussion.